packages/itwinui-css/src/utils/input-container/input-container.scss

```diff
@@ -186,10 +186,11 @@
 
   &.iui-actionable {
     align-items: center;
-    height: 90%;
+    height: calc(100% - 4px);
```

Comment:
comment for 4px might be useful

Reply:
I'm subtracting 2px on both sides to avoid overlapping with border/outline

Added comment